week3/BusinessCard/app/src/main/java/com/manuflowers/businesscard/SettingsFragment.kt

```diff
@@ -0,0 +1,84 @@
+package com.manuflowers.businesscard
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.os.Build
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.RadioGroup
+import androidx.appcompat.app.AppCompatDelegate
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.fragment.app.DialogFragment
+import com.manuflowers.businesscard.utils.Constants.PREFERENCES_MODE
+import com.manuflowers.businesscard.utils.Constants.THEME_MODE_KEY
+import com.manuflowers.businesscard.utils.ThemeMode
+import kotlinx.android.synthetic.main.fragment_settings.*
+
+class SettingsFragment : DialogFragment(), RadioGroup.OnCheckedChangeListener {
+
+    lateinit var sharedPreferences: SharedPreferences
```

Comment:
pay attention to warnings in the IDE. Keep your code squeaky clean. Here it's advising you to scope this var as private.

Reply:
Done